ds3-autogen-c/src/main/java/com/spectralogic/ds3autogen/c/converters/HeaderConverter.java

```diff
@@ -23,10 +23,14 @@
 import com.spectralogic.ds3autogen.c.models.Header;
 import com.spectralogic.ds3autogen.c.models.Request;
 import com.spectralogic.ds3autogen.c.models.Struct;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 import java.text.ParseException;
 
 public class HeaderConverter {
+    private static final Logger LOG = LoggerFactory.getLogger(HeaderConverter.class);
```

Comment:
It was used during development, but no longer... I'd still hesitate to remove it